vault/resource_jwt_auth_backend_role_test.go

```diff
@@ -548,6 +568,7 @@ resource "vault_jwt_auth_backend_role" "role" {
 	role_type = "jwt"
 
   bound_audiences = ["https://myco.test"]
+  bound_claims_type = "string"
```

Comment:
I think if `bound_claims` isn't being set, then `bound_claims_type` shouldn't be set either.
```suggestion
```